heft-plugins/heft-webpack5-plugin/src/WebpackConfigurationLoader.ts

```diff
@@ -76,16 +109,15 @@ export class WebpackConfigurationLoader {
     }
   }
 
-  private static _tryLoadWebpackConfiguration(
-    buildFolder: string,
-    configurationFilename: string
-  ): IWebpackConfigJs | undefined {
-    const fullWebpackConfigPath: string = path.join(buildFolder, configurationFilename);
-    if (FileSystem.exists(fullWebpackConfigPath)) {
+  private async _tryLoadWebpackConfigurationInnerAsync(
+    configurationPath: string
+  ): Promise<IWebpackConfigJs | undefined> {
+    const configExists: boolean = await FileSystem.existsAsync(configurationPath);
+    if (configExists) {
       try {
-        return require(fullWebpackConfigPath);
+        return await import(configurationPath);
```

Comment:
Does this throw an error that can be identified by `FileSystem.isNotExistError` if it doesn't exist?

Reply:
No, but it does throw a different error `ERR_MODULE_NOT_FOUND`. I can check for that.